src/cloud-api-adaptor/podvm/qcow2/build-s390x-se-image.sh

```diff
@@ -16,14 +16,44 @@ for i in /tmp/files/*.crt; do
     host_keys+="-k ${i} "
 done
 [[ -z $host_keys ]] && echo "Didn't find host key files, please download host key files to 'files' folder " && exit 1
-echo "Installing jq"
-export DEBIAN_FRONTEND=noninteractive
-sudo apt-get update > /dev/null 2>&1
-sudo apt-get install jq -y > /dev/null 2>&1
-sudo apt-get remove unattended-upgrades -y
-sudo apt-get autoremove
-sudo apt-get clean
-sudo rm -rf /var/lib/apt/lists/*
+
+if [ "${PODVM_DISTRO}" = "rhel" ]; then
+    export LANG=C.UTF-8
+    # Below is the tmp work-around to install cryptsetup on rhel9.4 and below s390x version base images,
+    #due to the issue : https://gitlab.com/qemu-project/qemu/-/issues/2054
+    cp /tmp/files/cryptsetup /usr/bin/cryptsetup
```

Comment:
the cryptsetup already be copied to `/usr/local/bin` so L24 and L25 need be removed.
otherwise will meet this error:
```
==> qemu.rhel: cp: cannot stat '/tmp/files/cryptsetup': No such file or directory
==> qemu.rhel: chmod: cannot access '/usr/bin/cryptsetup': No such file or directory
==> qemu.rhel: cryptsetup is required but it's not installed. Installing now...
==> qemu.rhel: Failed to install cryptsetup. Aborting.
=
```

Reply:
Add a new line L22:
```
    export PATH=$PATH:/usr/local/bin
```
and replace all `sudo cryptsetup` to `sudo -E cryptsetup`

Reply:
or maybe `sudo cp /usr/local/bin/cryptsetup /usr/bin`

Reply:
As per @snir911 suggestion here https://github.com/confidential-containers/cloud-api-adaptor/pull/1924#discussion_r1694838770 i have modified code to avoid extra step of copying. if we are adding this extra copy then ill revert back to my original code?  please suggest. 
Which is like 

Makefile does cp /usr/sbin/cryptsetup ./files;
& 
build-s390x-se-image.sh does  cp /tmp/files/cryptsetup /usr/bin/cryptsetup

Reply:
Hi @liudalibj, i've changed code back to original format post that I was able to build image. Could you please review.

```
[root@hpscc02 seeluser]# podman images
REPOSITORY                                                 TAG         IMAGE ID      CREATED       SIZE
quay.io/confidential-containers/podvm-libvirt-rhel-s390x   sl-final    8368079c41af  10 hours ago  3.04 GB
<none>                                                     <none>      55398315189a  10 hours ago  14.5 GB
quay.io/confidential-containers/podvm-binaries-rhel-s390x  sl-final    da36a893ed7e  10 hours ago  396 MB
<none>                                                     <none>      84685cf5f221  10 hours ago  15.4 GB
quay.io/confidential-containers/podvm-builder-rhel         sl-final    109f2a1c4338  11 hours ago  7.1 GB
registry.access.redhat.com/ubi9/ubi                        9.4         c78647ae33b8  11 days ago   217 MB

```

with same se-enabled podvm qcow2 image was able to bring up workload 

```
[root@bastion-sl ~]# oc get pods -n default
NAME             READY   STATUS    RESTARTS   AGE
osc-caa-commit   1/1     Running   0          20m
[root@bastion-sl ~]# 

```

below are the logs from peerpod deamon for reference which says LaunchSecurityType: S390PV

```
2024/07/30 03:28:43 [adaptor/cloud/libvirt] LaunchSecurityType: S390PV
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Checking if instance (podvm-osc-caa-commit-3c11b24d) exists
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Uploaded volume key /var/lib/libvirt/images/test-se-anj-dir/podvm-osc-caa-commit-3c11b24d-root.qcow2
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Create cloudInit iso
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Uploading iso file: podvm-osc-caa-commit-3c11b24d-cloudinit.iso
2024/07/30 03:28:43 [adaptor/cloud/libvirt] 47104 bytes uploaded
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Volume ID: /var/lib/libvirt/images/test-se-anj-dir/podvm-osc-caa-commit-3c11b24d-cloudinit.iso
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Create XML for 'podvm-osc-caa-commit-3c11b24d'
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Creating VM 'podvm-osc-caa-commit-3c11b24d'
2024/07/30 03:28:43 [adaptor/cloud/libvirt] Starting VM 'podvm-osc-caa-commit-3c11b24d'
2024/07/30 03:28:43 [adaptor/cloud/libvirt] VM id 1070
2024/07/30 03:29:05 [adaptor/cloud/libvirt] Instance created successfully
2024/07/30 03:29:05 [adaptor/cloud/libvirt] created an instance podvm-osc-caa-commit-3c11b24d for sandbox 3c11b24dca6d9ca7076eed23915ec8a1b15eb4b2a3d5f992b40ad2a5ec9c462c
2024/07/30 03:29:05 [util/k8sops] osc-caa-commit is now owning a PeerPod object
```